python/cudf/dataframe/column.py

```diff
@@ -568,3 +568,9 @@ def as_mask(self):
         device array
         """
         return cudautils.compact_mask_bytes(self.to_gpu_array())
+
+    @ioutils.doc_to_dlpack()
+    def to_dlpack(self):
+        """{docstring}"""
```

Comment:
This is cool! Is this the first time you are using this pattern?

Reply:
Other io functions use this pattern as well.